symplyphysics/laws/thermodynamics/heat_capacity_via_molar_heat_capacity.py

```diff
@@ -9,40 +9,40 @@
 )
 
 # Description
-## Molar heat capacity of a substance is its heat capacity divided by its amount in moles.
+## Heat capacity is molar heat capacity (heat capacity per unit amount of substance) times amount of substance.
 
-# Definition: C_m = C / n
-## C_m - molar heat capacity
+# Definition: C = C_m * n
```

Comment:
Now it is not a definition.
Also I disagree. It is more logical to define molar heat capacity from heat capacity.

Reply:
That's why I moved it to laws (although I forgot to rename to "Law" in the file.) We need to find the extensive heat capacity much more often than the intensive ones.
Plus, the intention behind this was to make the thermal energy law agnostic to the type of heat capacity and use other laws to substitute the correct one.

Reply:
But we could derive this previous definition.

Reply:
You mean you want to have, say, both the `C = c * m` law and `c = C / m` definition?

Reply:
No, I mean only definition. Law can be found by solving for C.

Reply:
So, okay, you'd prefer to have the definition `c = C / m` to having the law `C = c * m`, am I right?

Reply:
Yes